sql/atxs/atxs_test.go

```diff
@@ -24,6 +24,12 @@ func TestMain(m *testing.M) {
 	os.Exit(res)
 }
 
+func checkAtxEqual(t *testing.T, this, that *types.VerifiedActivationTx) {
+	require.Equal(t, this.Received().UnixNano(), that.Received().UnixNano())
+	that.SetReceived(this.Received())
+	require.Equal(t, this, that)
+}
```

Comment:
Is this needed because of different time zones in the time stamp? If yes imo we should parse it with `time.Local` when reading from DB. The internal clock of the node also uses `time.Local`.

Reply:
yes. it's due to the time zone. 
if we special case this one (parsing it to time.Local), that means we need to do that for all the time output by clock. specifically LayerToTime(). and i think that's too big a change for the purpose. 

only during testing do we compare two ATXs to be equal.

Reply:
> The internal clock of the node also uses time.Local.

right. i was wrong. ok. i will parse it to local time.